app/webpack/observations/show/components/observation_field_value.jsx

```diff
@@ -132,12 +132,26 @@ class ObservationFieldValue extends React.Component {
     if ( ofv.user ) {
       addedBy = (
         <div className="added-by">
+          { ofv.user && (
+          <div>
           <div className="view">
             { I18n.t( "label_colon", { label: I18n.t( "added_by" ) } ) }
           </div>
           <a href={`/people/${ofv.user.login}`}>
             { ofv.user.login }
           </a>
+          </div>
+          ) }
+          { ofv.updater && (
+          <div>
+          <div className="view">
+            { I18n.t( "label_colon", { label: I18n.t( "last_edited_by" ) } ) }
+          </div>
+          <a href={`/people/${ofv.updater.login}`}>
+            { ofv.updater.login }
+          </a>
+          </div>
+          ) }
```

Comment:
If you've got eslint running, it should be alerting you about indentation rules here. At a minimum, the contents of these outer divs should be indented.